providers/base/bin/prime_offload_tester.py

```diff
@@ -0,0 +1,334 @@
+#!/usr/bin/env python3
+"""prime offload test module."""
```

Comment:
This should go after the header, and overall brings zero value. Same information could be deduced from the file name.